api/models/choices/status.py

```diff
@@ -33,6 +33,18 @@ class CoordinatorStatus(models.TextChoices):
     REMOVED = "removed", "All access revoked, accounts closed"
     BANNED = "banned", "Banned from the project"
 
+    @classmethod
+    def active_statuses(cls) -> list["CoordinatorStatus"]:
+        return [
+            cls.PENDING,
+            cls.ONBOARDING,
+            cls.ONBOARDING_STALE,
+            cls.WORKING_BELOW_THRESHOLD,
+            cls.WORKING_OK,
+            cls.WORKING_LIMIT_REACHED,
+            cls.FINISHED_LEFT,
```

Comment:
Not sure if thats correct?

Reply:
I mean just the one on line 45, sounds like if a coordinator has that status, they arent really part of the project anymore

Reply:
Yep, I'm adding these notes 👌🏻